src/client.ts

```diff
@@ -0,0 +1,162 @@
+                .catch(reject)
+                .finally(() => clearTimeout(timeout));
+    });
+}
+
+export const getContainers = (con: Connection) => podmanJson(con, "libpod/containers/json", "GET", { all: true });
+
+export const streamContainerStats = (con: Connection, callback: MonitorCallback) =>
+    con.monitor(VERSION + "libpod/containers/stats", callback);
+
+export function inspectContainer(con: Connection, id: string) {
+    const options = {
+        size: false // set true to display filesystem usage
+    };
+    return podmanJson(con, "libpod/containers/" + id + "/json", "GET", options);
+}
+
+export const delContainer = (con: Connection, id: string, force: boolean) => podmanCall(con, "libpod/containers/" + id, "DELETE", { force });
+
+export const renameContainer = (con: Connection, id: string, config: JsonObject) => podmanCall(con, "libpod/containers/" + id + "/rename", "POST", config);
+
+export const createContainer = (con: Connection, config: JsonObject) => podmanJson(con, "libpod/containers/create", "POST", {}, JSON.stringify(config));
+
+export const commitContainer = (con: Connection, commitData: JsonObject) => podmanCall(con, "libpod/commit", "POST", commitData);
+
+export const postContainer = (con: Connection, action: string, id: string, args: JsonObject) => podmanCall(con, "libpod/containers/" + id + "/" + action, "POST", args);
+
+export const runHealthcheck = (con: Connection, id: string) => podmanCall(con, "libpod/containers/" + id + "/healthcheck", "GET", {});
+
+export const postPod = (con: Connection, action: string, id: string, args: JsonObject) => podmanCall(con, "libpod/pods/" + id + "/" + action, "POST", args);
+
+export const delPod = (con: Connection, id: string, force: boolean) => podmanCall(con, "libpod/pods/" + id, "DELETE", { force });
+
+export const createPod = (con: Connection, config: JsonObject) => podmanCall(con, "libpod/pods/create", "POST", {}, JSON.stringify(config));
+
+export function execContainer(con: Connection, id: string) {
+    const args = {
+        AttachStderr: true,
+        AttachStdout: true,
+        AttachStdin: true,
+        Tty: true,
+        Cmd: ["/bin/sh"],
+    };
+
+    return podmanJson(con, "libpod/containers/" + id + "/exec", "POST", {}, JSON.stringify(args));
+}
+
+export function resizeContainersTTY(con: Connection, id: string, exec: boolean, width: number, height: number) {
+    const args = {
+        h: height,
+        w: width,
+    };
+
+    let point = "containers/";
+    if (!exec)
+        point = "exec/";
+
+    return podmanCall(con, "libpod/" + point + id + "/resize", "POST", args);
+}
+
+function parseImageInfo(info: JsonObject): JsonObject {
+    const image: JsonObject = {};
+
+    if (info.Config) {
+        const config = info.Config as JsonObject;
+        image.Entrypoint = config.Entrypoint;
+        image.Command = config.Cmd;
+        image.Ports = Object.keys((config.ExposedPorts as JsonObject) || {});
+        image.Env = config.Env || [];
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2214-c7aa3398-20250801-083853-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)